pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
   "Programming Language :: Python :: 3.13",
 ]
 dependencies = [
-  "capellambse-context-diagrams>=0.7.3,<0.8",
+  "capellambse-context-diagrams>=0.7.8,<0.8",
```

Comment:
For the OperationalProcess context diagram, we actually need to bump to 0.7.9 now.